### ISSUE TITLE
 Immediately unlink and no output, and I have every necessary requirements

### ISSUE THREAD

**Issue by Strucker102** [hilts-vaughan/UnlinkMKV-GUI#32]
Hi! I have tried **UnlinkMKV-GUI 0.2.1** and **UnlinkMKV-GUI 0.1.4**, but none has worked for me. The "verbose output" it's empty everytime. When I press the Unlink button, immediately says that is done and the output folder is empty.

When I use the **0.2.1 version**, and check the "Native Backend" i got this output (photo):

Starting the processing of the file: C:\Users\Strucker\Desktop\test\[BB] Fullmetal Alchemist Brotherhood 01 [BDrip 720p] [510498C8].mkv

Starting the processing of the file: C:\Users\Strucker\Desktop\test\[BB] Fullmetal Alchemist Brotherhood Ending 1 [BDRip 720p] [0145C2D7].mkv

Starting the processing of the file: C:\Users\Strucker\Desktop\test\[BB] Fullmetal Alchemist Brotherhood Opening 1 [BDRip 720p] [49D246B8].mkv

But sadly, no output anyway...

I have Perl, MKVtoolnix and NET framework. And I did this in the command line:

Log::Log4perl
cpan XML::LibXML
cpan String:CRC32

What can I do? There is something I did wrong? Or I need some extra software? Please help me, I was trying unlink this anime for months.

Thanks!

![0](https://user-images.githubusercontent.com/68506737/87877377-45408380-c9de-11ea-91a9-710b5469588b.png)

**Comment by hilts-vaughan** [comment]
Have you tried the new version without the native backend? The native backend requires you to have MKVToolbox installed I believe. It's been some time since I worked on this.

If there is no output, I imagine it's because something low level went wrong. 

What is your system language? I recall there being problems for non-English users and i'm not sure that it was ever fixed.

**Comment by Strucker102** [comment]
Yes I try without the native backend, and no output anyway. My system language is Spanish :)

**Comment by hilts-vaughan** [comment]
You can try a couple things:

1. Set your System Language to English to see if that helps; I can try and patch the program later if it does but it's been a long time since I worked on this.

2. Make sure things on your PATH. In your command line, you can try running things like `mkvinfo` and make sure they pop up and run. If they don't, you might have to install some of the CLI utilties to get them running. 

3. If this fails, you can try running the Perl script from the "legacy" module to see if that works.

**Comment by Strucker102** [comment]
I do that, and same. Can I use UnlinkMKV in a Virtual Machine? I was thinking that maybe I can install linux in one.

On the other hand, MKVtoolnix can't unlink without anything else? Maybe adding all linked parts?

Anyway, thanks a lot for the help! Sorry if I bothered you. Have a nice day :)

**Comment by hilts-vaughan** [comment]
You can use it in a VM; I do all my development for this version in a VM since my primary OS is Linux.

I would check all the paths to make sure they are accessible; something is not setup right I would imagine. 

You can email me if you need more help.
